server/src/test/java/io/spine/server/projection/ProjectionRepositoryTest.java

```diff
@@ -352,30 +376,39 @@ public void expose_read_and_write_methods_for_last_handled_event_timestamp() {
      * procedures is exposed.
      */
     @Test
-    public void crete_stream_query() {
+    @DisplayName("create stream query")
+    void createStreamQuery() {
         assertNotNull(repository().createStreamQuery());
     }
 
-    /**
-     * Ensures that {@link ProjectionRepository#getEventStore()} which is used by the catch-up
-     * functionality is exposed to the package.
-     */
-    @Test
-    public void expose_event_store_to_package() {
-        assertNotNull(repository().getEventStore());
-    }
+    @Nested
+    @DisplayName("provide package-private access to")
+    class ExposeToPackage {
+
+        /**
+         * Ensures that {@link ProjectionRepository#getEventStore()} which is used by the catch-up
+         * functionality is exposed to the package.
+         */
+        @Test
+        @DisplayName("event store")
+        void eventStore() {
+            assertNotNull(repository().getEventStore());
+        }
 
-    /**
-     * Ensures that {@link ProjectionRepository#boundedContext()} which is used by the catch-up
-     * functionality is exposed to the package.
-     */
-    @Test
-    public void expose_bounded_context_to_package() {
-        assertNotNull(repository().boundedContext());
+        /**
+         * Ensures that {@link ProjectionRepository#boundedContext()} which is used by the catch-up
+         * functionality is exposed to the package.
+         */
+        @Test
+        @DisplayName("bounded context")
+        void boundedContext() {
+            assertNotNull(repository().boundedContext());
+        }
     }
 
     @Test
-    public void throw_exception_on_attempt_to_register_in_bc_with_no_messages_handled() {
+    @DisplayName("throw exception on attempt to register in BC with no messages handled")
```

Comment:
In fact there is a mistake in the method name. Projections subscribe to messages. We use `to handle` in relation to commands only.

Let's rename the method and fix the display name accordingly.